ObstacleAvoidance sketch

tarikdurakovic opened the pull request.

tarikdurakovic committed:
Delete ObstacleAvoidance.ino

tarikdurakovic committed:
Add files via upload

platisd commented on Obstacle_Avoidance.ino:
```diff
@@ -0,0 +1,47 @@
+    car.setSpeed(0);
+  }
+
+  delay(100);
```
Why `delay`? This can throw off your `car.update()`! If you want to decrease the refresh rate you should use `millis` and intervals instead of blocking calls. :)

tarikdurakovic replied:
I updated the sketch, please check it out

tarikdurakovic committed:
Update Obstacle_Avoidance.ino

I was testing the ultrasonic sensors because they weren't working so I forgot to delete delay(100). Started using SmartCar now as well.

platisd commented:
Don't forget to submit this in Canvas too please! :+1:
